api/config/cdr_versions_local.json

```diff
@@ -1,42 +1,50 @@
 [
   {
     "cdrVersionId": 1,
-    "name": "Synthetic Dataset v1 (Archived)",
+    "name": "Synthetic Dataset v1",
     "dataAccessLevel": 1,
-    "archivalStatus": 1,
     "bigqueryProject": "fc-aou-cdr-synth-test",
-    "bigqueryDataset": "SR2019q4r3",
+    "bigqueryDataset": "synthetic_cdr20180606",
     "creationTime": "2018-06-06 00:00:00Z",
     "releaseNumber": 1,
     "numParticipants": 946237,
-    "cdrDbName": "cdr",
-    "elasticIndexBaseName": "cdr"
+    "cdrDbName": "synth_r_2020q1_3",
```

Comment:
This PR is to exactly match test env.  Should I update test env to match the old "cdr" ?

Reply:
The MySQL databases in local and test are different, so that's not going to be possible. This is entirely an internal configuration detail which tells the server which MySQL database to connect to - if Puppeteer is failing on account of then some level of abstraction has been horribly broken.

Reply:
I see - I'll revert to "cdr"